src/packages/verticalsteps/doc.md

```
@@ -0,0 +1,99 @@
+| title | 流程步骤的标题 | `string` | `-` |
+| description | 流程步骤的描述性文字 | `string` | `-` |
+| icon | 图标(来自Icon组件的name属性) | `ReactNode` | `-` |
+| value | 流程步骤的索引 | `number` | `0` |
+| description | 流程步骤的描述性文字的html结构 | `React.ReactNode` | `-` |
+| onStepClick | 点击步骤的标题或图标时触发 | `(index: number) => void` | `-` |
+
+## 主题定制
+
+### 样式变量
+
+组件提供了下列 CSS 变量，可用于自定义样式，使用方法请参考 [ConfigProvider 组件](#/zh-CN/component/configprovider)。
```

Markdownlint

Link fragments should be valid

(MD051, link-fragments)